src/transformers/trainer.py

```diff
@@ -3725,6 +3733,9 @@ def training_step(
                 torch.npu.empty_cache()
             elif is_torch_mps_available(min_version="2.0"):
                 torch.mps.empty_cache()
+            elif is_torch_hpu_available():
+                # torch.hpu.empty_cache()
+                pass
```

Comment:
Similarly to the accelerate side, we should make a note of how this doesn't exist rn